src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  const totalCost = days * rentPrice;
+  const threeDays = 3;
+  const sevenDays = 7;
+
+  if (days >= sevenDays) {
+    return totalCost - bigDiscount;
+  } else if (days >= threeDays) {
+    return totalCost - smallDiscount;
+  } else {
+    return totalCost;
+  }
```

Comment:
```suggestion
  if (days >= sevenDays) {
    return totalCost - bigDiscount;
  }
  
  if (days >= threeDays) {
    return totalCost - smallDiscount;
  }
  
  return totalCost;
```
Do you really need to use if...else here?
You code will not go further after `return` anyway